File: Ejercicios en clases/poo java/Two java/library/Square.java
public class Square extends GeometricFigure {
    private double side;

    public Square(double side) {
        super("Square");
        this.side = side;
    }

    public double getSide() {
        return side;
    }

    @Override
    public double calculateArea() {
        return side * side;
    }
}

File: Ejercicios en clases/poo php/Two php/Library/square.php
<?php

class Square extends geometricFigures {
     private float $side;
     private float $calculate;
 
     public function __construct(float $side) {
         parent::__construct("Square");
         $this->side = $side;
     }
 
     public function getSide(): float {
         return $this->side;
     }
 
     public function calculateArea(): float {
         $this->calculate = $this->getSide() * $this->getSide();
         return $this->calculate;
     }
 }
 

 
